wsmatrix@martin.zurowietz.de/ThumbnailsBoxOverride.js

```diff
@@ -71,37 +86,37 @@ var ThumbnailsBoxOverride = class {
    _activateThumbnailAtPoint(stageX, stageY, time) {
       let [r, x, y] = this.transform_stage_point(stageX, stageY);
 
-      for (let i = 0; i < this._thumbnails.length; i++) {
-         let thumbnail = this._thumbnails[i]
-         let [w, h] = thumbnail.actor.get_transformed_size();
-
-         // Add x range check to check for both vertical and horizontal position
-         if (y >= thumbnail.actor.y && y <= thumbnail.actor.y + h &&
-            x >= thumbnail.actor.x && x <= thumbnail.actor.x + w) {
-            thumbnail.activate(time);
-            break;
-         }
-      }
-      this.queue_relayout();
+      let thumbnail = this._thumbnails.find(t => {
+         let [w, h] = t.get_transformed_size();
+         return y >= t.y && y <= t.y + h && x >= t.x && x <= t.x + w;
+      });
+      if (thumbnail)
+         thumbnail.activate(time);
```

Comment:
Please add enclosing `{}` for the `if` (even if it has only one statement).

Reply:
done